docs/02-lyric.md

```diff
@@ -0,0 +1,61 @@
+# Lyric
+
+Lyric is a tabular data management service designed to handle structured clinical and research data. Built on top of Lectern's dictionary framework, it provides a system for organizations to submit, validate, and manage structured data according to predefined schemas. While primarily used for clinical data management, Lyric's architecture remains domain-agnostic, allowing it to handle any type of structured data that can be defined within a Lectern dictionary.
+
+## Key Features
+
+- **Schema-driven validation:** Integrates with Lectern dictionaries to enforce data structure and relationships, validating TSV submissions against predefined schemas.
```

Comment:
"uses" isntead of "integrate" is probably simpler and _more_ correct

Reply:
expanding to by default support tsv and csv, so maybe just "validating submissions"? or tabular data files?